### DISCUSSION TITLE
Option for tag changes not to update the "modified" field.

### DISCUSSION THREAD

**Discussion by amreus** [TiddlyWiki/TiddlyWiki5#8782] [Ideas]
maybe this had been discussed  before. 

in some of my wikis tags are used only for organization.  When tags are changed then the tiddler shows in the Recent list.  This is a little misleading if what i am interested in is the text content modification time. 

Perhaps there could be an option to enable and disable updating the modified field per wiki for tag changes.

**Comment by Jermolene** [comment]
Hi @amreus there is a button in the sidebar tools menu that can be used to temporarily disable the updating of timestamps:

<img width="307" alt="image" src="https://github.com/user-attachments/assets/72030d91-532f-4488-9ffc-58ccdf51519e">
<img width="304" alt="image" src="https://github.com/user-attachments/assets/cfb2b101-30df-4179-a25e-8636498e0a14">

Note that this setting doesn't just affect tag changes, it affects all updates to tiddlers.

**Reply by amreus** [reply]
Yes thanks Jeremy.  I do use the button but it has proved too difficult for me to remember to use it when i need it.
